nix/checks/core.nix

```diff
@@ -75,10 +75,13 @@ in
           inherit inputs;
         };
         imports = [
-          ../nixos-configurations/core-master/configuration.nix
           inputs.self.nixosModules.default
         ];
 
+        scale-network.services.keaMaster.enable = true;
```

Comment:
Any objections to something like this? Its easier and less prone to side effects when we need to add specific network config to: `nixos-configurations/core-master/configuration.nix` or similar roles

Reply:
only hesitation would be that it makes it easier to drift between prod and test. But not a big concern

Reply:
I object to clustering Kea in the first place, but that ship has sailed. This seems fine though.

Reply:
Clustering Kea isnt actually enabled at the moment, we just run both as `master` and they are independent of one another

Reply:
ah my confusion, mixed it up with BIND.

Reply:
I thought the same thing when I picked this code up from 21x